tokio-postgres/src/client.rs

```diff
@@ -3,6 +3,7 @@ use crate::config::{Host, SslMode};
 use crate::connection::{Request, RequestMessages};
 use crate::copy_out::CopyOutStream;
 use crate::query::RowStream;
+use crate::replication::ReplicationStream;
```

Comment:
Should we have separate `LogicalReplicationStream` and `PhysicalReplicationStream`?